stack_definition.json

```diff
@@ -293,15 +289,15 @@
     },
     {
       "name": "3b - Databases for Elasticsearch",
-      "version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.f4b9793a-611c-4f6c-9181-ca1fbdaa37fa-global",
+      "version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.9e63c855-8749-412e-bc3e-bd24caaadc91-global",
       "inputs": [
         {
           "name": "use_existing_resource_group",
           "value": true
         },
         {
           "name": "resource_group_name",
-          "value": "ref:../../members/1 - Account Infrastructure Base/outputs/workload_resource_group_name"
+          "value": "ref:../../members/1 - Account Infrastructure Base/outputs/observability_resource_group_name"
```

Comment:
The elastic deployment should stay in workloads RG. I am sorry I was not specific in my last comment - only Event Notifications should be moved to observability (which you did).